mlflow/gateway/constants.py

```diff
@@ -0,0 +1,6 @@
+GATEWAY_VERSION = "0.1.0"
```

Comment:
Why do we need to version the Gateway differently than the rest of MLflow? IMO, using the MLflow version is easier for users to reason about

Reply:
great point. pulling from mlflow.version.VERSION instead :)